### PR TITLE
Issue 179

### PR DESCRIPTION
**Pull request by gdey**
This fixes some additional rendering artifacts.

### CODE REVIEW

File: util/convert/convert.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+package convert
```
Comment (ARolek) [inline util/convert/convert.go]: move this to an internal package.

File: geom/encoding/wkt/internal/token/token.go [hunk 2]
```diff
@@ -257,36 +237,58 @@ func (t *T) ParseMultiPoint() (pts geom.MultiPoint, err error) {
 	for {
 		t.EatSpace()
 		// Grab the sub points. Need to check to see if there is a (
```
[the remark this answers is not in the document]
Comment (gdey) [reply geom/encoding/wkt/internal/token/token.go]: The thing we are checking for is '('. I know it looks incomplete.

[the remark this answers is not in the document]
Comment (gdey) [reply geom/encoding/wkt/internal/token/token.go]: Subpoints are: of the form:
XXX YYY ZZZ MMM,
or 
( XXX YYY ZZZ MMM) 
so we need to see if we have a starting '(' for the sub point.

[the remark this answers is not in the document]
Comment (ARolek) [reply geom/encoding/wkt/internal/token/token.go]: got it.

**Pull request merged**